test/specs/collections/Breadcrumb/BreadcrumbDivider-test.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import BreadcrumbDivider from 'src/collections/Breadcrumb/BreadcrumbDivider'
+import * as common from 'test/specs/commonTests'
+
+describe('BreadcrumbDivider', () => {
+  common.isConformant(BreadcrumbDivider)
+  common.rendersChildren(BreadcrumbDivider)
+
+  it('renders as a div by default', () => {
+    shallow(<BreadcrumbDivider />)
+      .should.have.tagName('div')
+  })
+
+  describe('icon prop', () => {
```

Comment:
Yes, it's good solution, but seems it will not work without #279.

![image](https://cloud.githubusercontent.com/assets/14183168/16680711/499ad192-44f9-11e6-98fa-d027e981bbe4.png)